### PR TITLE
fix(deps): fnv1a included which supports Safari

### PR DESCRIPTION
**Pull request by galmail**
### What does this PR do?

This PR fix [issue 496](https://github.com/nearform/graphql-hooks/issues/496) which supports Safari

### Checklist

- [ ] I have checked the [contributing document](../blob/master/CONTRIBUTING.md)
- [ ] I have added or updated any relevant documentation
- [x] I have added or updated any relevant tests

### CODE REVIEW

File: packages/graphql-hooks-memcache/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphql-hooks-memcache",
-  "version": "1.3.2",
```
Comment (mcollina) [inline packages/graphql-hooks-memcache/package.json:3]: Please revert this. I think this might go in some other release.

**Comment by osdevisnot** [comment]
@mcollina - this is impacting usage of memcache for safari.

Can we cut a release with this fix?

**Comment by chughes87** [comment]
@mcollina my team is waiting for this to be released. Could this be done sometime soon or should we look into alternative solutions?

**Comment by mcollina** [comment]
I’ll get to it as soon as I can, it has been a very busy month

**Comment by chughes87** [comment]
Great thank you I appreciate the update.

**Comment by mcollina** [comment]
Shipped.

**Pull request merged**